metadata.yaml

```diff
@@ -15,6 +15,10 @@ provides:
     interface: k8s-service
     schema: https://raw.githubusercontent.com/canonical/operator-schemas/master/k8s-service.yaml
     versions: [v1]
+  metrics-endpoint:
+    interface: prometheus_scrape
+  grafana-dashboard:
```

Comment:
```suggestion
  grafana-dashboards:
```

Reply:
Shall we also change this in the [docs](https://github.com/canonical/grafana-k8s-operator#to-bundle-dashboards-as-part-of-your-charm)? It says we should just add `grafana-dashboard`.

Reply:
@simskij unfortunately that piece of docu is wrong, it mistakes the relation name for the interface